templates/secret.yaml

```diff
@@ -8,79 +8,55 @@ metadata:
   {{- include "plausible-analytics.labels" . | nindent 4 }}
 type: Opaque
 data:
-  {{- if .Values.adminUser.email }}
-  ADMIN_USER_EMAIL: {{ .Values.adminUser.email | toString | b64enc }}
+  {{- if .Values.secretKeyBase }}
+  SECRET_KEY_BASE: {{ .Values.secretKeyBase | toString | b64enc }}
+  {{- else }}
+  SECRET_KEY_BASE: {{ randAlphaNum 90 | toString | b64enc }}
   {{- end }}
-  {{- if .Values.adminUser.password }}
-  ADMIN_USER_PWD: {{ .Values.adminUser.password | toString | b64enc }}
+  {{- if .Values.databaseURL }}
+  DATABASE_URL: {{ .Values.databaseURL | toString | b64enc }}
   {{- end }}
-  {{- if .Values.adminUser.name }}
-  ADMIN_USER_NAME: {{ .Values.adminUser.name | toString | b64enc }}
+  {{- if .Values.clickhouseDatabaseURL }}
+  CLICKHOUSE_DATABASE_URL: {{ .Values.clickhouseDatabaseURL | toString | b64enc }}
   {{- end }}
-  {{- if .Values.postgresql.url }}
-  DATABASE_URL: {{ .Values.postgresql.url | toString | b64enc }}
+  {{- if .Values.mailer.enabled }}
+  {{- if eq .Values.mailer.adapter "Bamboo.SMTPAdapter" }}
```

Comment:
Why for ?
The https://hexdocs.pm/bamboo_smtp/Bamboo.SMTPAdapter.html use smtp but all the others rely on third party API that only need an api key except mailgun.
The only commong things to mailer are the email which is the sender.
My update on mailer values reflect that.
```
mailer:
  enabled: false  # Enable/Disable functionality
  email: ""  # the email address of the email sender
  adapter: ""  # "Bamboo.SMTPAdapter", "Bamboo.MailgunAdapter", "Bamboo.MandrillAdapter", "Bamboo.SendGridAdapter"
  smtp:
    host: ""  # The host address of your smtp server.
    port: ""  # The port of your smtp server.
    username: ""  # The username/email in case SMTP auth is enabled.
    password: ""  # The password in case SMTP auth is enabled.
    ssl: ""  # If SSL is enabled for SMTP connection
    retries: ""  # Number of retries to make until mailer gives up.
  mailgun:
    apiKey: ""
    domain: ""
    baseURI: ""
  postmarkApiKey: ""
  mandrillApiKey: ""
  sendgridApiKey: ""

```
Did I miss something ?

Reply:
https://github.com/pablo-co/bamboo_postmark for reference only require the postmarkApiKey

Reply:
@mrdotb Nope sorry, I misunderstood when I read the docs from plausible. Thanks for the link, you are right! Let's merge :rocket: